docs/metrics-events.md

```diff
@@ -194,19 +194,15 @@ in the preceding five days.
 
 ## Significant changes
 
-### Train 73
+### Train 74
 
-* Expiry time
-  for metrics context data in memcached
-  was increased from 30 minutes
-  to 2 hours.
-
-* The `flow.complete` event
+* Flow event data validation
   was implemented.
 
-* The `${viewName}` part
-  of `flow.${viewName}.begin`
-  and `flow.${viewName}.engage`
+* The `${viewName}` part of
```

Comment:
That's only on the `attempt` event, which I've mentioned separately a few bullets down the list.